### ISSUE TITLE
add convenience drop_sweeps function to radar class

### ISSUE THREAD

**Issue by nguy** [ARM-DOE/pyart#652]
Per conversation on the [mailing list](https://groups.google.com/forum/#!topic/pyart-users/DKH1eYdIPuM), it might be nice to have a convenient way to remove sweeps from a radar volume.

This may also be of interest in Issue #649 ?

**Comment by scollis** [comment]
yep.. And fairly easy to do... Sounds like a good student project.. I am going to make this medium priority.
